libs/domains/organization/src/lib/slices/cluster/cluster.slice.ts

```diff
@@ -86,7 +87,7 @@ export const editClusterAdvancedSettings = createAsyncThunk<
     settings: ClusterAdvancedSettings
     toasterCallback: () => void
   }
->('cluster/advancedSettings/edit', async (data) => {
+>('cluster/advanced-settings/edit', async (data) => {
```

Comment:
The camelCase notation is the one used by nx when it autogenerate your slice, this is why we adopted this notation at first. You want to go to kebab-case?
<img width="758" alt="CleanShot 2023-02-07 at 12 06 38@2x" src="https://user-images.githubusercontent.com/6163954/217228432-4f9bd3cb-7bb3-473d-9bd8-a92c8353aec8.png">